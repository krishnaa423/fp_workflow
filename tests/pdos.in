&PROJWFC
outdir='./tmp'
prefix='struct'
filpdos='struct_pdos.dat'
/
