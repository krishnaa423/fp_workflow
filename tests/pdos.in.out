
     Program PROJWFC v.7.2 starts on 17Jul2024 at  5:10:23 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes
     28146 MiB available memory on the printing compute node when the environment starts
 

     Reading xml data from directory:

     ./tmp/struct.save/

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want

 
     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         295     295     91                 3287     3287     609
 
     Using Pencil Decomposition
 

     Gaussian broadening (default values): ngauss,degauss=   0    0.000735


     Calling projwave .... 
     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
  Problem Sizes 
  natomwfc =             8
  nbnd     =            14
  nkstot   =             3
  npwx     =           412
  nkb      =            16
 

     Atomic states used for projection
     (read from pseudopotential files):

     state #   1: atom   1 (Si ), wfc  1 (l=0 m= 1)
     state #   2: atom   1 (Si ), wfc  2 (l=1 m= 1)
     state #   3: atom   1 (Si ), wfc  2 (l=1 m= 2)
     state #   4: atom   1 (Si ), wfc  2 (l=1 m= 3)
     state #   5: atom   2 (Si ), wfc  1 (l=0 m= 1)
     state #   6: atom   2 (Si ), wfc  2 (l=1 m= 1)
     state #   7: atom   2 (Si ), wfc  2 (l=1 m= 2)
     state #   8: atom   2 (Si ), wfc  2 (l=1 m= 3)

 k =   0.0000000000  0.0000000000  0.0000000000
==== e(   1) =    -5.49024 eV ==== 
     psi = 0.498*[#   1]+0.498*[#   5]
    |psi|^2 = 0.997
==== e(   2) =     6.55050 eV ==== 
     psi = 0.160*[#   2]+0.160*[#   3]+0.160*[#   4]+0.160*[#   6]+0.160*[#   7]
          +0.160*[#   8]
    |psi|^2 = 0.962
==== e(   3) =     6.55050 eV ==== 
     psi = 0.160*[#   2]+0.160*[#   3]+0.160*[#   4]+0.160*[#   6]+0.160*[#   7]
          +0.160*[#   8]
    |psi|^2 = 0.962
==== e(   4) =     6.55050 eV ==== 
     psi = 0.160*[#   2]+0.160*[#   3]+0.160*[#   4]+0.160*[#   6]+0.160*[#   7]
          +0.160*[#   8]
    |psi|^2 = 0.962
==== e(   5) =     8.99022 eV ==== 
     psi = 0.161*[#   2]+0.161*[#   3]+0.161*[#   4]+0.161*[#   6]+0.161*[#   7]
          +0.161*[#   8]
    |psi|^2 = 0.966
==== e(   6) =     8.99022 eV ==== 
     psi = 0.161*[#   2]+0.161*[#   3]+0.161*[#   4]+0.161*[#   6]+0.161*[#   7]
          +0.161*[#   8]
    |psi|^2 = 0.966
==== e(   7) =     8.99022 eV ==== 
     psi = 0.161*[#   2]+0.161*[#   3]+0.161*[#   4]+0.161*[#   6]+0.161*[#   7]
          +0.161*[#   8]
    |psi|^2 = 0.966
==== e(   8) =     9.88950 eV ==== 
     psi = 0.493*[#   1]+0.493*[#   5]
    |psi|^2 = 0.985
==== e(   9) =    14.19300 eV ==== 
 
    |psi|^2 = 0.000
==== e(  10) =    14.19300 eV ==== 
 
    |psi|^2 = 0.000
==== e(  11) =    14.47557 eV ==== 
     psi = 0.001*[#   1]+0.001*[#   5]
    |psi|^2 = 0.002
==== e(  12) =    17.65479 eV ==== 
     psi = 0.004*[#   2]+0.004*[#   3]+0.004*[#   4]+0.004*[#   6]+0.004*[#   7]
          +0.004*[#   8]
    |psi|^2 = 0.024
==== e(  13) =    17.65479 eV ==== 
     psi = 0.004*[#   2]+0.004*[#   3]+0.004*[#   4]+0.004*[#   6]+0.004*[#   7]
          +0.004*[#   8]
    |psi|^2 = 0.024
==== e(  14) =    17.65479 eV ==== 
     psi = 0.004*[#   2]+0.004*[#   3]+0.004*[#   4]+0.004*[#   6]+0.004*[#   7]
          +0.004*[#   8]
    |psi|^2 = 0.024

 k =  -0.3535533906  0.3535533906 -0.3535533906
==== e(   1) =    -3.13810 eV ==== 
     psi = 0.383*[#   1]+0.383*[#   5]+0.039*[#   2]+0.039*[#   3]+0.039*[#   4]
          +0.039*[#   6]+0.039*[#   7]+0.039*[#   8]
    |psi|^2 = 0.997
==== e(   2) =    -0.50924 eV ==== 
     psi = 0.127*[#   2]+0.127*[#   3]+0.127*[#   4]+0.127*[#   6]+0.127*[#   7]
          +0.127*[#   8]+0.113*[#   1]+0.113*[#   5]
    |psi|^2 = 0.987
==== e(   3) =     5.31137 eV ==== 
     psi = 0.165*[#   2]+0.165*[#   3]+0.165*[#   4]+0.165*[#   6]+0.165*[#   7]
          +0.165*[#   8]
    |psi|^2 = 0.989
==== e(   4) =     5.31137 eV ==== 
     psi = 0.165*[#   2]+0.165*[#   3]+0.165*[#   4]+0.165*[#   6]+0.165*[#   7]
          +0.165*[#   8]
    |psi|^2 = 0.989
==== e(   5) =     8.03725 eV ==== 
     psi = 0.347*[#   1]+0.347*[#   5]+0.038*[#   2]+0.038*[#   3]+0.038*[#   4]
          +0.038*[#   6]+0.038*[#   7]+0.038*[#   8]
    |psi|^2 = 0.921
==== e(   6) =     9.76673 eV ==== 
     psi = 0.128*[#   2]+0.128*[#   3]+0.128*[#   4]+0.128*[#   6]+0.128*[#   7]
          +0.128*[#   8]
    |psi|^2 = 0.765
==== e(   7) =     9.76673 eV ==== 
     psi = 0.128*[#   2]+0.128*[#   3]+0.128*[#   4]+0.128*[#   6]+0.128*[#   7]
          +0.128*[#   8]
    |psi|^2 = 0.765
==== e(   8) =    14.17630 eV ==== 
     psi = 0.023*[#   2]+0.023*[#   3]+0.023*[#   4]+0.023*[#   6]+0.023*[#   7]
          +0.023*[#   8]+0.019*[#   1]+0.019*[#   5]
    |psi|^2 = 0.174
==== e(   9) =    17.10490 eV ==== 
 
    |psi|^2 = 0.005
==== e(  10) =    17.10490 eV ==== 
 
    |psi|^2 = 0.005
==== e(  11) =    17.82758 eV ==== 
     psi = 0.032*[#   2]+0.032*[#   3]+0.032*[#   4]+0.032*[#   6]+0.032*[#   7]
          +0.032*[#   8]
    |psi|^2 = 0.192
==== e(  12) =    17.82758 eV ==== 
     psi = 0.032*[#   2]+0.032*[#   3]+0.032*[#   4]+0.032*[#   6]+0.032*[#   7]
          +0.032*[#   8]
    |psi|^2 = 0.192
==== e(  13) =    17.94337 eV ==== 
     psi = 0.030*[#   1]+0.030*[#   5]
    |psi|^2 = 0.061
==== e(  14) =    19.52849 eV ==== 
     psi = 0.101*[#   2]+0.101*[#   3]+0.101*[#   4]+0.101*[#   6]+0.101*[#   7]
          +0.101*[#   8]+0.092*[#   1]+0.092*[#   5]
    |psi|^2 = 0.792

 k =   0.0000000000  0.0000000000 -0.7071067812
==== e(   1) =    -1.32226 eV ==== 
     psi = 0.275*[#   1]+0.275*[#   5]+0.074*[#   2]+0.074*[#   3]+0.074*[#   4]
          +0.074*[#   6]+0.074*[#   7]+0.074*[#   8]
    |psi|^2 = 0.993
==== e(   2) =    -1.32226 eV ==== 
     psi = 0.275*[#   1]+0.275*[#   5]+0.074*[#   2]+0.074*[#   3]+0.074*[#   4]
          +0.074*[#   6]+0.074*[#   7]+0.074*[#   8]
    |psi|^2 = 0.993
==== e(   3) =     3.61772 eV ==== 
     psi = 0.166*[#   2]+0.166*[#   3]+0.166*[#   4]+0.166*[#   6]+0.166*[#   7]
          +0.166*[#   8]
    |psi|^2 = 0.996
==== e(   4) =     3.61772 eV ==== 
     psi = 0.166*[#   2]+0.166*[#   3]+0.166*[#   4]+0.166*[#   6]+0.166*[#   7]
          +0.166*[#   8]
    |psi|^2 = 0.996
==== e(   5) =     7.14434 eV ==== 
     psi = 0.176*[#   1]+0.176*[#   5]+0.080*[#   2]+0.080*[#   3]+0.080*[#   4]
          +0.080*[#   6]+0.080*[#   7]+0.080*[#   8]
    |psi|^2 = 0.835
==== e(   6) =     7.14434 eV ==== 
     psi = 0.176*[#   1]+0.176*[#   5]+0.080*[#   2]+0.080*[#   3]+0.080*[#   4]
          +0.080*[#   6]+0.080*[#   7]+0.080*[#   8]
    |psi|^2 = 0.835
==== e(   7) =    16.61306 eV ==== 
     psi = 0.154*[#   2]+0.154*[#   3]+0.154*[#   4]+0.154*[#   6]+0.154*[#   7]
          +0.154*[#   8]
    |psi|^2 = 0.924
==== e(   8) =    16.61306 eV ==== 
     psi = 0.154*[#   2]+0.154*[#   3]+0.154*[#   4]+0.154*[#   6]+0.154*[#   7]
          +0.154*[#   8]
    |psi|^2 = 0.924
==== e(   9) =    17.62601 eV ==== 
 
    |psi|^2 = 0.004
==== e(  10) =    17.62601 eV ==== 
 
    |psi|^2 = 0.004
==== e(  11) =    19.05155 eV ==== 
 
    |psi|^2 = 0.000
==== e(  12) =    19.05155 eV ==== 
 
    |psi|^2 = 0.000
==== e(  13) =    19.60212 eV ==== 
     psi = 0.042*[#   1]+0.042*[#   5]+0.008*[#   2]+0.008*[#   3]+0.008*[#   4]
          +0.008*[#   6]+0.008*[#   7]+0.008*[#   8]
    |psi|^2 = 0.134
==== e(  14) =    19.60212 eV ==== 
     psi = 0.042*[#   1]+0.042*[#   5]+0.008*[#   2]+0.008*[#   3]+0.008*[#   4]
          +0.008*[#   6]+0.008*[#   7]+0.008*[#   8]
    |psi|^2 = 0.134

Lowdin Charges: 

     Atom #   1: total charge =   3.9585, s =  1.0324, 
     Atom #   1: total charge =   3.9585, p =  2.9261, pz=  0.9754, px=  0.9754, py=  0.9754, 
     Atom #   2: total charge =   3.9585, s =  1.0324, 
     Atom #   2: total charge =   3.9585, p =  2.9261, pz=  0.9754, px=  0.9754, py=  0.9754, 
     Spilling Parameter:   0.0104
 
     PROJWFC      :      0.15s CPU      0.18s WALL

 
   This run was terminated on:   5:10:23  17Jul2024            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
Warning: ieee_invalid is signaling
Warning: ieee_divide_by_zero is signaling
Warning: ieee_inexact is signaling
FORTRAN STOP
